src/FakeItEasy/Configuration/ArgumentCollection.cs

```diff
@@ -68,6 +69,8 @@ public int Count
         /// </summary>
         public IEnumerable<string> ArgumentNames { get; }
 
+        internal MethodInfo Method { get; }

```

Comment:
It feels a little weird to have a `Method` live on an `ArgumentCollection`. But I guess if I ask myself, "arguments for what?", it seems okay.